<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalyticsDashboardClass</class>
 <widget class="QWidget" name="AnalyticsDashboardClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnalyticsDashboard</string>
  </property>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
background-color:#CCD0CD;

}
QLabel{
font-size:20px;
}</string>
   </property>
   <widget class="QLabel" name="label_118">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font-size:34px;
font-weight:bold;
color:#2B2A2A;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/Assets/analysis.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>731</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>369</y>
      <width>791</width>
      <height>381</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>789</width>
       <height>379</height>
      </rect>
     </property>
     <widget class="QFrame" name="frame_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>741</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
  border-radius: 25px;
  border: 2px solid white ;
background-color:white;

}
QPushButton{
  border-radius: 10px;
  background: #699CCC;
color:white;



}
QPushButton:hover{
background: #CCD0CD;
color:Black;
}

#pushButton_2{
background: #FF7F7F;
}
#pushButton_2:hover{
background: #CCD0CD;
color:Black;
}
QLabel{
font-size:15px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QLabel" name="label_119">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#label_2{color:black;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="Resource.qrc">:/Assets/home128.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_120">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Location :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_121">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>100</y>
         <width>201</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Space :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_122">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>140</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Bedrooms :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_123">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>60</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Price : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_124">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>100</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Type :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_125">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>140</y>
         <width>331</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Building Number: 17  </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_126">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>180</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>User ID :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_127">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>180</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>User Name :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_128">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Property ID : </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_129">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>20</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Apartmant number :</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_130">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
font-size:34px;
font-weight:bold;
color:#2B2A2A;
}</string>
    </property>
    <property name="text">
     <string>Analytics</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_131">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Users :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_132">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/Assets/group.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_133">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Properties :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_134">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/Assets/properties.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_135">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Top Compared Properties </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_136">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/Assets/top_compare.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>393</x>
      <y>120</y>
      <width>20</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_17">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
  border-radius: 15px;
  border: 2px solid white ;
background-color:white;

}
QLabel
{
font-size:15px;
font-weight:bold;
color:#2B2A2A;
}
QPushButton{
  border-radius: 10px;
  background: #FF7F7F;
color:white;



}

QPushButton:hover{
background: #CCD0CD;
color:Black;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="label_137">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>3</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>#12345678901234</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_138">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>3</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Ahmed</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_139">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>5</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>Assets/profile.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_140">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>User with most listings </string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
